images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -110,7 +110,7 @@ $toolsList = @(
     (Get-GitLFSVersion),
     (Get-GitFTPVersion),
     (Get-HavegedVersion),
-    (Get-HerokuVersion),
+    "$(Get-HerokuVersion) $(Get-AptSourceRepository -PackageName 'heroku')",
```

Comment:
I'd suggest appending apt source inside the `Get-HerokuVersion`, because we already use the package name there, so we can avoid hard-coding it in the common generator. Also, Generator code looks more clear the way it is now.

Reply:
Done